src/wp-includes/class-wp-block-supports.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Block support flags.
+ *
+ * @package WordPress
+ * 
+ * @since 5.6.0
+ */
+
+/**
+ * Class encapsulating and implementing Block Supports.
+ *
+ * @access private
+ */
+class WP_Block_Supports {
```

Comment:
Are there unit tests planned for this class? cc. @youknowriad @mcsf

Reply:
There is actually a unit test in Gutenberg, we should bring it here.